module/documents/actor/actor.mjs

```diff
@@ -209,6 +209,20 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
     return data;
   }
 
+  /* -------------------------------------------- */
+
+  /**
+   * Is this actor under the effect of this property from some status?
+   * @param {string} key      A key in `DND5E.conditionProperties`.
+   * @returns {boolean}       Whether the actor is affected.
+   */
+  hasCondition(key) {
```

Comment:
I'm going to vote on a slightly longer name here because this is checking for specific effects imposed by conditions, not the conditions themselves.

```suggestion
  hasConditionEffect(key) {
```